lib/node_modules/@stdlib/repl/lib/commands/themes.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var format = require( '@stdlib/string/format' );
+var log = require( './../log.js' );
+
+
+// MAIN //
+
+/**
+* Returns a callback to be invoked upon calling the `themes` command.
+*
+* @private
+* @param {REPL} repl - REPL instance
+* @returns {Function} callback
+*/
+function command( repl ) {
+	return onCommand;
+
+	/**
+	* Gets available color themes for syntax-highlighting.
+	*
+	* @private
+	* @returns {void}
+	*/
+	function onCommand() {
+		var output;
+		var themes;
+		var i;
+
+		themes = repl.themes();
+		if ( themes.length === 0 ) {
+			log( repl, 'No available themes.' );
+		}
+		output = format( 'Available themes: `%s`', themes[ 0 ] );
+		for ( i = 1; i < themes.length; i++ ) {
+			output += format( ', `%s`', themes[ i ] );
+		}
+		output += '.';
```

Comment:
It is not clear why this loop is necessary.

```js
output = format( 'Available themes: `%s`', themes.join( ', ' ) );
```

Reply:
I instead decided to just return the array so it becomes more analogous to `workspaces()`.

Reply:
That should be fine, so long as we return a copy of the theme list and not a reference to the underlying array.